### PR TITLE
[KEYCLOAK-6301] Disable service account after disabling in Client

### PR DESCRIPTION
**Pull request by douglaspalmer**

### CODE REVIEW

**Comment by stianst** [comment]
I can't see how this fixes the problem. This seems to only change the field of service account on the JSON returned.

What I think we need is that when someone changes the client to disabled we should disable the associated service account user. Then re-enable it if the client is re-enabled.

**Comment by stianst** [comment]
We also need a test that gets a refresh token for the client, then disables the client, then tries to refresh the token.

File: services/src/main/java/org/keycloak/services/resources/admin/ClientResource.java [hunk 1]
```diff
@@ -25,17 +25,7 @@
...
-import org.keycloak.models.ClientTemplateModel;
-import org.keycloak.models.Constants;
-import org.keycloak.models.KeycloakSession;
-import org.keycloak.models.ModelDuplicateException;
-import org.keycloak.models.RealmModel;
-import org.keycloak.models.RoleModel;
-import org.keycloak.models.UserCredentialModel;
-import org.keycloak.models.UserModel;
-import org.keycloak.models.UserSessionModel;
+import org.keycloak.models.*;
```
Comment (stianst) [inline services/src/main/java/org/keycloak/services/resources/admin/ClientResource.java]: Please don't use * in imports.

**Review by stianst — CHANGES_REQUESTED** [review]
Looks good. One minor thing is to not use * in imports.

**Comment by douglaspalmer** [comment]
Sorry, I’ll fix that, Intellij did it automatically and I accepted it.

Regards
Doug

**Review by stianst — APPROVED** [review]

**Pull request merged**